tests/scripts/shepherd/update-gomod-shepherd.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+if [ "$1" == "-h" ] || [ -z "$1" ]; then
+  echo "Usage: `basename $0` <path-to-upstream-shepherd> <shepherd-release-branch> <path-to-rancher>" 
+  echo "WARN: this will update your local go.mod and go.sum files in your rancher repo."
+  printf "Requirements:\n* local clone of upstream rancher/shepherd\n* clone of rancher fork with local changes\n"
+  echo "example: `basename $0` ~/upstream-shepherd release/v2.8 ~/rancher-fork"
+  exit 0
+fi
+
+echo "getting latest shepherd version for branch $2"
+cd $1
+git checkout $2 -q
+git fetch -q
+git pull -q
+export SHEPHERD_VERSION=$(curl -s https://proxy.golang.org/github.com/rancher/shepherd/@v/$(git log -n 1 --pretty=format:"%H").info | grep -E -o "\bv0.0.0-+[A-Za-z0-9.-]+[A-Za-z0-9.-]\b")
+echo $SHEPHERD_VERSION
+
+echo "writing version to go.mod, then tidying in $3"
+cd $3
+sed -E -i 's/\bgithub.com\/rancher\/shepherd v0.0.0-+[A-Za-z0-9.-]+[A-Za-z0-9.-]\b/github.com\/rancher\/shepherd '"$SHEPHERD_VERSION"'/g' go.mod
```

Comment:
Instead of doing this why don't you do `go get github.com/rancher/shepherd@version` I'm a little iffy about the sed edit.

Reply:
But I'm good either way tbh